lib/tdf3/src/client/index.ts

```diff
@@ -352,7 +350,7 @@ export class Client {
     scope = { attributes: [], dissem: [] },
     autoconfigure,
     source,
-    asHtml = false,
```

Comment:
why remove the false default?

Reply:
We no longer need to assert it is `=== false` as that is the default (and I think has been for some time)